web_client/templates/annotationListWidget.pug

```diff
@@ -0,0 +1,40 @@
+.g-annotation-list-header
+  i.icon-pencil
+  | Annotations
+
+if annotations.length
```

Comment:
I think it would be better to not show the entire annotation list block when there aren't any annotations.